### ISSUE TITLE
small documentation issue: MISO and MOSI pins swapped

### ISSUE THREAD

**Issue by tingox** [hlide/MZ-SD2CMT#2]
It looks like the documentation has a small issue: on the Mega2560 MISO in pin 50 and MOSI is pin 51, according to "Connections" on this page https://www.arduino.cc/en/Reference/SPI

when I fixed that, my SDcard started working. :)

**Comment by hlide** [comment]
Oh, I just realized that project is too old compared to the SHARPENTIERS/MZ-SD2CMT so you'd better got it instead of this one.

**Comment by tingox** [comment]
Actually, I did get that, but for some reason I couldn't open an issue there, so I did it here instead. :)

**Comment by hlide** [comment]
Yeah, I've also just noticed it so an issue tab is now available in the SHARPENTIERS/MZ-SD2CMT.
